editor/editor_export_godot3.cpp

```diff
@@ -698,7 +873,8 @@ Error EditorExportGodot3::_get_property_as_text(const Variant &p_variant, String
 				p_string += "ExtResource(" + str.get_slice(":", 1) + ")";
 			} else {
 
-				str = "\"" + str.c_escape_multiline() + "\"";
+				// Call _replace_resource in case it's a path to a scene/resource
+				str = "\"" + _replace_resource(str).c_escape_multiline() + "\"";
```

Comment:
To be on the safe side, in case some users saved paths to XML resources or scenes in their Globals.